global.ini

```diff
@@ -64318,7 +64318,7 @@ refinery_ui_ProcessingType_SlowWasteful=Kazen Winnowing
 refinery_ui_ProcessingType_SlowWasteful_Desc=Trotz seiner mäßigen Verarbeitungsgeschwindigkeit und relativ geringen Ausbeute wird das Kazen-Winden, das im 22. Jahrhundert entwickelt wurde, aufgrund seiner relativ einfachen Einrichtung und seines niedrigeren Preispunkts immer noch häufig verwendet. Sobald das Rohmaterial seine Flüssigkeitstemperatur erreicht, wird hochdruckiges Gas eingeführt und verwendet, um verbleibende Verunreinigungen zu winden.
 refinery_ui_ProcessingType_SlowWasteful_Details=Moderate Geschwindigkeit // Geringe Kosten // Geringe Ausbeute
 refinery_ui_Profile=Profil
-refinery_ui_RefinementCenter=Raffinerie Center
+refinery_ui_RefinementCenter=Refinement Center
```

Comment:
@MaxM1211 ich habe den Namen mal zurückgesetzt. Ich denke das zählt ja auch zu Eigennamen und in Missionstexten sind solche Namen auch im Original.